cmd/erasure-object.go

```diff
@@ -1499,7 +1499,6 @@ func (er erasureObjects) restoreTransitionedObject(ctx context.Context, bucket s
 	}
 
 	_, err = er.CompleteMultipartUpload(ctx, bucket, object, uploadID, uploadedParts, ObjectOptions{
-		VersionID:        oi.VersionID,
```

Comment:
Unintentional change? Doesn't seem related.

Reply:
It is intentional - it is superfluous and not needed.

Reply:
But I can remove it from this PR.